Dockerfile

```diff
@@ -12,11 +12,11 @@ RUN dotnet publish -c Release -r linux-musl-x64 -o /opt/analyzer
 
 # Create analyze script
 RUN mkdir /opt/analyzer/bin && \
-    echo "/opt/analyzer/Exercism.Analyzers.CSharp \$1 \$2" > /opt/analyzer/bin/analyze.sh && \
+    printf "#!/bin/sh\n\nexport DOTNET_SYSTEM_GLOBALIZATION_INVARIANT=true\n/opt/analyzer/Exercism.Analyzers.CSharp \$1 \$2\n" >/opt/analyzer/bin/analyze.sh && \
```

Comment:
Did you find a resource that describes how this environment variable helps?

Reply:
Only that https://github.com/dotnet/coreclr/issues/14927 identifies this as a way of setting `System.Globalization.Invariant` without needing to play with the dotnet env

Reply:
I would have expected something like this: ENV COMPlus_EnableDiagnostics=0

See this thread: https://github.com/dotnet/docs/issues/10217

I'd be totally fine with adding that to the Dockerfile.

Reply:
I switched to `printf` because `echo` (in that underlying distro, or shell) didn't expand `\n`